packages/sdk/app-framework/src/vite-plugin/boot-loader/boot-loader.css

```
@@ -0,0 +1,117 @@
+  height: 2px;
+  overflow: hidden;
+  background: rgba(127, 127, 127, 0.18);
+  border-radius: 1px;
+}
+
+#boot-loader-bar::before {
+  content: '';
+  position: absolute;
+  inset: 0;
+  width: 40%;
+  background: currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)